notebooks/264-speculative-sampling/264-speculative-sampling.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.                if np.random.random() &lt; min(1, (q_item / p_item)):  # accepted
</pre><p><code>q_item</code> is logits, <code>p_item</code> is probability after softmax. You should apply softmax to <code>q</code> first.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>